core/src/main/java/io/grpc/ClientCall.java

```diff
@@ -97,7 +97,7 @@
  * @param <ReqT> type of message sent one or more times to the server.
  * @param <RespT> type of message received one or more times from the server.
  */
-@DoNotMock("Use InProcessTransport and make a fake server instead")
+@DoNotMock("Use InProcessServerBuilder and make a full featured in process server instead")
```

Comment:
The "full featured" wording is taken from the javadoc of `InProcessServerBuilder`, as I didn't want to use the word "fake" which feels contradictory. I'll use a more neutral word to avoid this altogether.